index.js

```diff
@@ -1,10 +1,40 @@
 openNav = () => {
-  document.getElementById(
-    'myNav',
-  ).style.width = '100%';
+  document.getElementById('myNav').style.width = '100%';
 };
+
 closeNav = () => {
-  document.getElementById(
-    'myNav',
-  ).style.width = '0%';
-};
\ No newline at end of file
+  document.getElementById('myNav').style.width = '0%';
+};
+const project = {
+  name: 'portfolio',
+  description:
+    'Personal portfolio that contains project information, skills, and contact',
+  technologies: ['HTML', 'CSS', 'JavaScript'],
+  liveVersion: 'https://enixword.github.io/My_Portfolio/ ',
+  sourceCode: 'https://github.com/EnixWord/My_Portfolio',
+};
+
+const buttonPopup = document.querySelector('button');
+const modalPopup = document.querySelector('.block-popup');
+function Display() {
+  modalPopup.style.display = 'block';
+}
+function closePopup() {
+  modalPopup.style.display = 'none';
+}
+
+// ------------------contact form validation -----------------
+
+function validateEmail() {
+  const email = document.getElementById('email').value;
+  const error = document.getElementById('errorMessage');
+  if (email.toLowerCase() !== email) {
+    error.innerHTML = 'Please enter your email in lowercase';
```

Comment:
- [ **OPTIONAL** ] Good job so far, your app is working as expected. But, I think it would be better you change the color of the error message text. As it is an error message, `red` color will look better than white.